stream_alert/rule_processor/config.py

```diff
@@ -100,21 +102,18 @@ def load_env(context):
         context: The AWS Lambda context object.
 
     Returns:
-        {'lambda_region': 'region_name',
-         'account_id': <ACCOUNT_ID>,
-         'lambda_function_name': 'function_name',
-         'lambda_alias': 'qualifier'}
+        [dict] including the following items:
```

Comment:
lol not fixing this now since this is a common thing across the code base right now and I don't want to introduce inconsistency. When we go to support the napoleon stuff we should update all of these in one fell swoop